gammapy/image/profile.py

```diff
@@ -194,6 +196,241 @@ def save(self, filename):
         raise NotImplementedError
 
 
+class ImageProfile(object):
+    """
+    Image profile class.
+
+    The image profile data is stored in `~astropy.table.Table` object, with the
+    following columns:
+
+        * `x_ref` Coordinate bin center (required). 
+        * `x_min` Coordinate bin minimum (optional).
+        * `x_max` Coordinate bin maximum (optional).
+        * `profile` Image profile data (required).
+        * `profile_err` Image profile data error (optional).
+
+    """
+    def __init__(self, table):
+        self.table = table
+
+    def smooth(self, kernel='box', radius=0.1 * u.deg, **kwargs):
+        """
+        Smooth profile with error propagation.
+
+        Smoothing is described by a convolution:
+
+        .. math::
+
+                x_j = \sum_i x_{(j - i)} h_i
+
+        Where :math:h_i are coefficients of the convolution kernel.
```

Comment:
Does this work? I think backticks are needed, no?
```
:math:`h_i`
```
Is it really useful to give the general formula how 1D convolution works?
Why not just say "convolution"?